lib/routes/run.js

```diff
@@ -1,20 +1,63 @@
 'use strict';
 var chalk = require('chalk');
+var findUp = require('find-up');
+var inquirer = require('inquirer');
 var namespaceToName = require('yeoman-environment').namespaceToName;
+var path = require('path');
+var userHome = require('user-home');
+
+// Checks if .yo-rc.json exists in parent directory
+var filePath = findUp.sync('.yo-rc.json');
+if (filePath !== null) {
+  filePath = path.dirname(filePath);
+}
+
+function scaffoldInto(app, name) {
+  var baseName = namespaceToName(name);
+  inquirer.prompt([{
+    name: 'scaffold',
+    type: 'confirm',
+    message: 'Are you sure you want to scaffold in ' + filePath +
+      ' ?'
+  }], function (answer) {
+    if (answer.scaffold) {
+      console.log(
```

Comment:
move 24-34 into its own function so it can be reused here and at lien 51. DRY is nice.

Maybe add a param to control the console output since it seems like it shouldn't be outputted here as the user clearly indicated this is where they want to scaffold to, unlike on like 51 where they haven't given that consent yet. Perhaps in this case, display the notice in line 36 so they know what to do next.